src/App.js

```diff
@@ -1,27 +1,11 @@
-import logo from './logo.svg';
+import 'bootstrap/dist/css/bootstrap.min.css';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
```

Comment:
- [ ] This component `App` should be class-based, kindly convert the function component into a class component, this is a requirement
e.g: 
```
class App extends Component {
  render() {
    return {
      <Calculator />
    };
  }
}

```